src/test/java/uk/gov/cshr/controller/signup/SignupControllerTest.java

```diff
@@ -78,8 +87,8 @@ public void shouldFailValidationIfEmailAddressNotInWhitelist() throws Exception
                 post("/signup/request")
```

Comment:
You should probably be adding a mock for       
`when(whiteListRepository.existsByDomain(DOMAIN)).thenReturn(false);`

The only reason this is false is because the test is making a db call and that db doesnt contain the value.

Reply:
This value does exist in DB. will discuss